### PR TITLE
Api messages translation

### PR DESCRIPTION
**Pull request by FrancoisPog**
## Changes
#181 
As we have decided that the API language is French, the messages from the controllers are translated into French.

### CODE REVIEW

File: server/controllers/moleculesImporter.js [hunk 1]
```diff
@@ -129,14 +129,14 @@ async function importMolecules(req, res) {
...
-        message: "File imported",
+        message: "Fichier importé",
         warnings: [],
         imported: true,
       });
     }
   } catch (error) {
     if (HeaderErrors.isInstance(error)) {
-      res.sendUsageError(422, "Badly formatted file", {
+      res.sendUsageError(422, "Fichier mal formatté", {
```
Comment (nathanaelhoun) [inline server/controllers/moleculesImporter.js:139]:
```suggestion
      res.sendUsageError(422, "Fichier mal formaté", {
```

File: server/controllers/question.js [hunk 2]
```diff
@@ -136,7 +136,7 @@ async function generateQuestion(req, res) {
     });
   } catch (error) {
     if (NotEnoughDataError.isInstance(error)) {
-      res.sendUsageError(422, "Not enough data to generate this type of question", {
+      res.sendUsageError(422, "Pas assez de données sont disponible pour ce type de question", {
```
Comment (nathanaelhoun) [inline server/controllers/question.js:139]:
```suggestion
      res.sendUsageError(422, "Pas assez de données disponibles pour ce type de question", {
```

File: server/controllers/user.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ async function login(req, res) {
       isAdmin,
     });
   } else {
-    res.sendUsageError(401, "Authentication failed");
+    res.sendUsageError(401, "Echec de l'authentification");
```
Comment (nathanaelhoun) [inline server/controllers/user.js:58]:
```suggestion
    res.sendUsageError(401, "Échec de l'authentification");
```

File: server/controllers/usersImportation.js [hunk 4]
```diff
@@ -94,21 +94,21 @@ async function importUsers(req, res) {
...
-        message: "File tested but not imported",
+        message: "Fichié testé mais pas importé",
         warnings,
         imported: false,
       });
     }
   } catch (error) {
     if (HeaderErrors.isInstance(error)) {
-      res.sendUsageError(422, "Badly formatted file", {
+      res.sendUsageError(422, "Fichier mal formatté", {
```
Comment (nathanaelhoun) [inline server/controllers/usersImportation.js:111]:
```suggestion
      res.sendUsageError(422, "Fichier mal formaté", {
```

**Review by nathanaelhoun — APPROVED** [review]
_Une bonne chose de faite !_

**Review by valentinperignon — APPROVED** [review]
SBPM

**Pull request merged**